### DISCUSSION TITLE
debug option to expose actual commands?

### DISCUSSION THREAD

**Discussion by mjm2312** [danitseitlin/redis-modules-sdk-ts#226] [Q&A]
Hi,

I'm trying to investigate whether multiple filters are supported in mrange. For example, I can run this command on the CLI:

TS.MRANGE 1676504220000 1676505900000 FILTER id=(id1, id2) type=aType prop=aProp GROUPBY prop REDUCE min

But i can't seem to pass a list of filters using mrange -- I've tried delimiting the filter string with a space as i've done above and with a comma, but neither work.

Is there a debug option that shows the exact command that's run against the database? showDebugOptions shows arguments but not that exact command

Thank you

**Comment by mjm2312** [comment]
It seems that when I pass `type=aType prop=aProp` as filter, it works, and so does `id=(id1,id2)`. But when i use `id=(id1,id2)` with either type or prop, it breaks

Is there a way to pass a list of conjunctive and disjunctive filters?

  Error: RedisTimeSeries class (TS.MRANGE): ReplyError: ERR TSDB: failed parsing labels

**Reply by danitseitlin** [reply]
@mjm2312 Hi, Thanks for the question. I will need to take a look and update you. Will update you ASAP.

**Reply by mjm2312** [reply]
Thanks a lot. For what it's worth, I can pass the list of filters I want ` id=(id1, id2) prop=Prop type=Type` with queryindex, another fn that allows a list of filters. I'm not sure if it's an issue with `redis-modules-sdk` or `ioredis`.

**Comment by mjm2312** [comment]
The issue has to do with how FILTERS gets passed to send command, will open PR with fix this or next week
